asab/zookeeper/container.py

```diff
@@ -63,3 +70,16 @@ async def get_data(self, child, encoding="utf-8"):
 
 	async def get_raw_data(self, child):
 		return await self.ZooKeeper.get_data("{}/{}".format(self.ZooKeeperPath, child))
+
+	@staticmethod
+	async def build_client(Config, z_url):
```

Comment:
No static method please, this functionality should be in `asab.zookeeper.builder` module in a function `build_client`